docs/articles/lavalink/intro.md

```diff
@@ -1,7 +1,85 @@
 # Lavalink - the newer, better way to do music
-Lavalink is a standalone program, written in Java, using JDA. It's a 
+[Lavalink](https://github.com/Frederikam/Lavalink) is a standalone program, written in Java, using JDA. It's a 
 lightweight solution for playing music from sources such as YouTube or 
 Soundcloud. Unlike raw voice solutions, such as VoiceNext, Lavalink can handle 
 hundreds of concurrent streams, and supports sharding.
 
-TODO
\ No newline at end of file
+## Configuring Java
+In order to run Lavalink, you must have Java 11 or greater installed.
```

Comment:
I believe the requirement might've been bumped to JRE 13 with recent releases. Worth noting that JRE 14 has fucked TLS, but generally I believe they're gonna go with latest available.